CoffeeChat/Controller/HomeViewController.swift

```diff
@@ -63,9 +63,12 @@ class HomeViewController: UIViewController {
         tabCollectionView.layer.shadowOpacity = 1
         tabCollectionView.layer.shadowRadius = 4
         view.addSubview(tabCollectionView)
-        
-        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
 
+        setUpConstraints()
+    }
+    
+    private func getUserMatching() {
+        guard let netId = UserDefaults.standard.string(forKey: Constants.UserDefaults.userNetId) else { return }
```

Comment:
possible refactor in the future, we write googleID and netID on the backend, not googleId / netId, to keep things consistent on both ends :)